src/App.vue

```diff
@@ -98,8 +98,11 @@
               <header>
                 <h1>{{ getFilingName }}</h1>
               </header>
+              <p class="mt-4 mb-6" v-if="getFilingSubtitle">
```

Comment:
In my experiene, it's easier to lay out whitespace if you only use top whitespace. (That way, each element only has to be concerned with the whitespace above it.) (There is an exception for the bottom element in some blocks.)

I believe the bottom margin here, and the top margin in the Stepper component below, will create too much whitespace.

If the intent is to create whitespace between the subtitle and the Signin component (assuming the Stepper is not rendered) then that should be handled either in the Stepper or in the Signin. This is not the subtitle's problem to worry about, and doing so complicates the layout and logic.

Reply:
Sorry, the Signin component doesn't render anything. I meant whatever component displays below the Stepper.